src/chart-review/chart-review.component.tsx

```diff
@@ -8,76 +8,86 @@ import Encounters from "./encounters/encounters.component";
 import Allergies from "./allergies/allergies.component";
 import Conditions from "./conditions/conditions.component";
 import Programs from "./programs/programs.component";
+import { useConfig } from "@openmrs/esm-module-config";
 
 export default function ChartReview(props: any) {
   const match = useRouteMatch();
   const location = useLocation();
 
   const { patientUuid } = useParams();
   const { widget } = useParams();
+  const config = useConfig();
 
-  const config = {
-    defaultPath: `/patient/${patientUuid}/chart/`,
-    defaultTabIndex: 0,
-    widgets: [
-      {
-        name: "Summaries",
-        path: `summaries`,
-        component: () => {
-          return <Summaries />;
-        }
-      },
-      {
-        name: "Results",
-        path: `results`,
-        component: () => {
-          return <Results />;
-        }
-      },
-      {
-        name: "Orders",
-        path: `orders`,
-        component: () => {
-          return <Orders />;
-        }
-      },
-      {
-        name: "Encounters",
-        path: `encounters`,
-        component: () => {
-          return <Encounters />;
-        }
-      },
-      {
-        name: "Conditions",
-        path: `conditions`,
-        component: () => {
-          return <Conditions />;
-        }
-      },
-      {
-        name: "Allergies",
-        path: `allergies`,
-        component: () => {
-          return <Allergies />;
-        }
-      },
-      {
-        name: "Programs",
-        path: `programs`,
-        component: () => {
-          return <Programs />;
-        }
+  const defaultPath = `/patient/${patientUuid}/chart/`;
+
+  function getConfigWidgets() {
+    const w = [];
+    config.widgets.map(widgetName => {
+      w.push(coreWidgets[widgetName]);
```

Comment:
map returns a new transformed array, no need of `w` here
```suggestion
return config.widgets.map(widgetName => coreWidgets[widgetName]);
```

Reply:
Done